controllers/categories.go

```diff
@@ -26,23 +27,28 @@ func (cc *categoryController) CreateCategory(ctx echo.Context) error {
 	var category models.Category
 	if err := ctx.Bind(&category); err != nil {
 		cc.logger.Error(err.Error())
+
 		return ctx.JSON(http.StatusBadRequest, echo.Map{"status": "invalid request"})
 	} else if cc.db.CheckCategoryExists(category.Name) {
+		log.Gl.Info("Category already exists")
 		return ctx.JSON(http.StatusConflict, echo.Map{"status": "category already exists"})
 	} else {
 		_, err := cc.db.CreateCategory(&category)
 		if err != nil {
 			cc.logger.Error(err.Error())
+			log.Gl.Info("Cannot create category")
 			ctx.JSON(http.StatusConflict, echo.Map{"status": "cannot create category"})
 		}
+		log.Gl.Info("Category created")
 		return ctx.JSON(http.StatusCreated, echo.Map{"status": "category created"})
 	}
 }
 
 func (cc *categoryController) GetCategory(ctx echo.Context) error {
 	category, err := cc.db.GetCategory(ctx.Param("name"))
 	if err != nil || category.ID == 0 {
-		cc.logger.Error(err.Error())
+
+		log.Gl.Info("Category not found")
```

Comment:
Log the error itself.
- not as an info
- log category.ID but make sure it won't panic.